### PR TITLE
media-video/tsmuxer: update to new version

### PR DESCRIPTION
**Pull request by winterheart**
Updated to opensourced (license is Apache-2.0) version available at https://github.com/justdan96/tsMuxer
Closes: https://bugs.gentoo.org/691814

Removed old version and license SmartLabs related to it.

### CODE REVIEW

**Comment by ajakk** [comment]
Can you fix the commit log to drop the merge commit? Ideally simply drop the merge commit and rebase the others onto current master.

**Comment by winterheart** [comment]
Please remove SmartLab license file after removing binary version.

File: media-video/tsmuxer/tsmuxer-2.6.16_p20220706.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (ajakk) [inline media-video/tsmuxer/tsmuxer-2.6.16_p20220706.ebuild:4]: I'm sorry it's taken so long to review this. Can you bump up to EAPI=8?

**Comment by ajakk** [comment]
Looks like there at least a missing dependency with USE=qt5 on `dev-qt/linguist-tools`:

```
CMake Error at /usr/lib64/cmake/Qt5/Qt5Config.cmake:28 (find_package):
  Could not find a package configuration file provided by "Qt5LinguistTools"
  with any of the following names:

    Qt5LinguistToolsConfig.cmake
    qt5linguisttools-config.cmake

  Add the installation prefix of "Qt5LinguistTools" to CMAKE_PREFIX_PATH or
  set "Qt5LinguistTools_DIR" to a directory containing one of the above
  files.  If "Qt5LinguistTools" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  tsMuxerGUI/CMakeLists.txt:17 (find_package)
```

If you don't have easy access to minimal environments for testing, `app-portage/iwdevtools` catches a lot of mismatched dependency issues.

File: media-video/tsmuxer/tsmuxer-2.6.16_p20220706.ebuild (hunk 1 above)
Comment (winterheart) [reply media-video/tsmuxer/tsmuxer-2.6.16_p20220706.ebuild]: Done

**Comment by ajakk** [comment]
Please note that you should use `Bug:` instead of `Closes:` as security handles closing security bugs, I went ahead and did the switch on merge. Thanks for your work!

**Pull request closed without merging**